libraries/Bluefruit52Lib/src/services/BLEHidGeneric.cpp

```diff
@@ -245,6 +245,13 @@ err_t BLEHidGeneric::begin(void)
   VERIFY_STATUS( _chr_control.begin() );
   _chr_control.write8(0);
 
+  // HID Gamepad
+  _chr_gamepad.setProperties(CHR_PROPS_READ | CHR_PROPS_NOTIFY);
+  _chr_gamepad.setFixedLen(sizeof(hid_gamepad_report_t));
+  _chr_gamepad.setPermission(SECMODE_ENC_NO_MITM, SECMODE_NO_ACCESS);
+  VERIFY_STATUS(_chr_gamepad.begin());
+  _chr_gamepad.write8(0);
+
```

Comment:
Not required as characteristic is defined within extended loop above (_num_input = 4)